installation/before-install.md

```diff
@@ -55,5 +55,21 @@ Use `sysctl -p` command or reboot your node for the changes to take effect.
 
 These kernel options were originally taken from the presentation [How Netflix Tunes EC2 Instances for Performance](https://www.slideshare.net/brendangregg/how-netflix-tunes-ec2-instances-for-performance) by [Brendan Gregg](http://www.brendangregg.com/), Senior Performance Architect at AWS re:Invent 2017.
 
-If this article is incorrect or outdated, or omits critical information, please [let us know](https://github.com/fluent/fluentd-docs-gitbook/issues?state=open). [Fluentd](http://www.fluentd.org/) is an open-source project under [Cloud Native Computing Foundation \(CNCF\)](https://cncf.io/). All components are available under the Apache 2 License.
+## Use sticky bit symlink/hardlink protection
+
+Fluentd sometimes uses predictable paths for dumping, writing files, and so on.
+For symlink attack protection, check the following parameters are set up as `1`:
+
+```text
+fs.protected_hardlinks = 1
+fs.protected_symlinks = 1
+```
 
+This settings are almost enough for time-of-check to time-of-use (TOCTOU, TOCTTOU or TOC/TOU) which is a class of software bugs.
+
+CentOS 7 or later, Ubuntu 18.04 (bionic) or later, and Debian GNU/Linux 10 (buster) or later are supported these parameters.
+If you truned off these protection, please turn on them.
```

Comment:
typo? turned off

Reply:
I'd fixed them in https://github.com/fluent/fluentd-docs-gitbook/pull/318/commits/a1c3d075161434eaa7c09232202dada0daf22a1f and https://github.com/fluent/fluentd-docs-gitbook/pull/318/commits/c7e652e2c46004c86d9978b1bf288ae3385eb483.